src/components/Loading.js

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact'
+import { useSoftkey, useI18n } from 'hooks'
+export const Loading = ({ close, message }) => {
+  const i18n = useI18n()
+
+  const onClose = () => {
+    if (close) {
+      close()
+    } else {
+      history.back()
+    }
+  }
+
+  useSoftkey('Loading', {
+    right: i18n.i18n('softkey-close'),
+    onKeyRight: onClose
+  })
+
+  return <div class='loading-planet'>
```

Comment:
There should be parenthesis around the JSX here.

Reply:
I'm not sure what it means here, but this returns a single <div/>, that's fine wihout having the parenthesis here, is there any code styling concern here?

Reply:
This is for consistency with the other components. Put `(` right after `return` and start the JSX on the next line. This way the markup is indented correctly.